sidm/configs/selections.yaml

```diff
@@ -99,6 +98,121 @@ base: &base
   evt_cuts: &evt_base
     - *evt_baseNoLj
     - ">=2 LJs"
+# Custom base channel inheriting from 'base'
```

Comment:
These new selections all look good, but would you mind moving them down lower somewhere in the yaml so the most commonly used ones stay near the top?